src/style.css

```diff
@@ -1,3 +1,47 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.logo__img {
+  display: inline-block;
+  position: absolute;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  position: absolute;
+  right: 50px;
+  height: 60px;
+  margin: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin: auto 20px;
+}
+
+.nav__link {
+  position: relative;
+  font-size: 12px;
```

Comment:
The font size of 12px might be too small for readability on some devices. Consider using a relative unit like `em` or `rem` to ensure better accessibility and responsiveness.